src/commands/reminder/command.ts

```diff
@@ -0,0 +1,79 @@
+import { addMilliseconds, formatDistanceToNow, isBefore } from "date-fns";
+import { TextChannel } from "discord.js";
+import parseDuration from "parse-duration";
+import { client } from "../..";
+import { DI } from "../../database";
+import { Reminder, User } from "../../database/entities";
+import { Command } from "../../types/Command";
+
+let reminderCache: Reminder[] = [];
+
+const reminderInterval = setInterval(() => {
```

Comment:
ideally when this command is not enabled, it should not be listening for reminders

Reply:
TRUE